test/TestServerProjectsLowLevel/paging/Generated/PagingClient.cs

```diff
@@ -60,7 +60,9 @@ public PagingClient(AzureKeyCredential credential, Uri endpoint, PagingClientOpt
         }
 
         /// <summary> A paging operation that must return result of the default &apos;value&apos; node. </summary>
-        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
+        /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
+        /// <returns> The <![CDATA[AsyncPageable<BinaryData>]]> from the service containing a list of <![CDATA[BinaryData]]> objects. Details of the body schema for each item in the collection are in the Remarks section below. </returns>
```

Comment:
We should not use CDATA.  This should only be used in rare circumstances where we cannot use standard tags.  If we cannot use the `<see/>` tag for these, let's remove those tags and file an issue to come back to it later.